tutorials/receiving-money.md

```diff
@@ -0,0 +1,275 @@
+```
+curl https://api.paystack.co/transaction/initialize \
+-H "Authorization: Bearer SECRET_KEY" \
+-H "Content-Type: application/json" \
+-d '{"amount": 500000, "email": "customer@email.com"}' \
+-X POST
+```
+Transactions can be initialised via different payment methods - Paystack Inline, Paystack Standard.
+
+### Payment Parameters
+
+The most important fields in the request body are the `amount` to be charged and the customer `email`.
+
+If your application does not require customer emails, you can generate an `email` for the transaction with any unique identifier from your system like phone number or customer id: 08001234567@yoursite.com or j239dj3hn9393@yourapp.com.
+
+The identifier should be unique to the customer for all their transactions so that you can track the customer transaction history on your Paystack dashboard.
+
+You can add a `reference` to the request body which you use to identify this transaction on your own system. If you don't send a reference, Paystack will generate one for you for that transaction. Also transaction references must be unique for each transaction, otherwise you will get a "Duplicate transaction" error.
+
+The `amount` values that you pass to Paystack must be in kobo. To convert a Naira value to kobo, multiply by 100. So if you wanted to charge N100, the amount value to send will be 10000.
+
+###Adding Extra Information to the Payment Request
+
+Sometimes you want to pass extra information to your transaction. The initialization request body accepts a `metadata` object in which you can add other information you want. You can pass the object directly into the `metadata`. You can also use `custom_fields` array within the metadata. Custom fields is an array of objects. The values you pass in the `custom_fields` array will displayed in the transaction details on the Paystack Dashboard and on the customer's receipt.
+
+A transaction initialization object with `metadata` will look like this:
+
+```JSON
+{
+  email: "me@you.us",
+  amount: 100000,
+  metadata: {
+    account_id: "i923d923",
+    reg_number: "101290",
+    custom_fields: [
+      {
+        display_name: "Favourite Movie",
+        variable_name: "favourite_movie",
+        value: "Pirates of the Carribeans"
+      },
+      {
+        display_name: "Worst Movie",
+        variable_name: "worst_movie",
+        value: "La la Land"
+      }
+    ]
+  }
+}
+```
+
+####Adding Custom Filters
+
+Custom Filters allow you to set some preferences on the checkout form. For example, you can decide to receive only certain card types on the **Pay With Card** channel. If you were building an application that requires recurring debit, you can choose to allow customers enter only cards that support recurring debit. For **Pay With Bank**, you can choose to accept only certain banks by passing thier bank codes ([here is how to get list of banks and their bank codes][https://developers.paystack.co/reference#list-banks]).
```

Comment:
...you can choose to accept only certain banks by passing **_their_** bank codes